blockly-ws-search/test/workspace_search_test.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Unit tests for WorkspaceSearch.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+const assert = require('assert');
+const Blockly = require('blockly');
+const sinon = require('sinon');
+
+const WorkspaceSearch = require('../dist/workspace-search.umd').WorkspaceSearch;
+
+suite('WorkspaceSearch', () => {
+  setup(() => {
+    this.jsdom = require('jsdom-global')();
+    this.blocklyDiv = document.createElement('div');
+    this.blocklyDiv.id = 'blocklyDiv';
+    document.body.appendChild(this.blocklyDiv);
+    this.workspace = Blockly.inject('blocklyDiv');
+    this.workspaceSearch = new WorkspaceSearch(this.workspace);
+  });
+
+  teardown(() => {
+    this.workspace.dispose();
```

Comment:
Can also add in the dispose for workspace search here

Reply:
I think the workspace dispose call might also not be necessary, because I call the cleanup method for the jsdom-global library (which is mocking all the DOM things). 
I'm going to remove the workspace dispose for now, since I think its also not needed, but can discuss id you disagree.